.github/ISSUE_TEMPLATE/regression.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Regression
+about: Something that used to work but doesn't anymore
```

Comment:
What's the difference between this and a bug, from an end user perspective?  Should this be something we ask them to mention in the bug report category?

Reply:
I think having this as an option would help reduce some work for us. If the user knows something worked beforehand they can categorize it correctly with this and we don't have to. If they don't know they can just call it a bug and we can change the label.
And I think separating bugs from regressions is good. If everything is just a bug it becomes kind of meaningless to even have that label.

Reply:
Sounds good to me!  We should probably clarify that in the bug report template as well, to avoid confusion.

Reply:
The

> about: Something that used to work but doesn't anymore

is visible when you select a template for your issue. I think that should be enough?

Reply:
I guess?  Personally my eyes might skip over that but if it's sufficiently obvious then it should be fine.